python/ray/dashboard/tests/node_stats_test.py

```diff
@@ -0,0 +1,33 @@
+from ray.dashboard.node_stats import NodeStats
+from ray.ray_constants import REDIS_DEFAULT_PASSWORD
+from datetime import datetime
+from time import sleep
+import pytest
+
+
+def test_basic(ray_start_with_dashboard):
+    """Dashboard test that starts a Ray cluster with a dashboard server running,
+    then hits the dashboard API and asserts that it receives sensible data."""
+    addr_info = ray_start_with_dashboard
+    redis_address = addr_info["redis_address"]
+    redis_password = REDIS_DEFAULT_PASSWORD
+    node_stats = NodeStats(redis_address, redis_password)
+    node_stats.start()
+    # Wait for node stats to fire up.
+    MAX_START_TIME_S = 15
```

Comment:
This seems destined to be flaky, at a minimum we should increase this timeout

Reply:
Ok